### PR TITLE
Updating azure-ml-cli version (1.33.0)

### PR DESCRIPTION
**Pull request by singankit**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [X] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [X] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [X] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
azure-ml-cli

File: src/index.json [hunk 1]
```diff
@@ -4712,7 +4712,8 @@
                         "Programming Language :: Python :: 3.5",
                         "Programming Language :: Python :: 3.6",
                         "Programming Language :: Python :: 3.7",
-                        "Programming Language :: Python :: 3.8"
+                        "Programming Language :: Python :: 3.8",
+                        "Programming Language :: Python :: 3.9"
```
Comment (harneetvirk) [inline src/index.json:4716]: Did we started supporting Python 3.9 for ml cli?

Comment (singankit) [reply src/index.json]: Thanks for catching it. Have made the necessary update.

**Comment by jiasli** [comment]
`azdev linter` fails:

https://dev.azure.com/azure-sdk/public/_build/results?buildId=1029389&view=logs&j=ccdac6d3-465a-5ada-8819-d5b4a6b3a631&t=ffdbba7d-e523-5908-c7bf-0703bafb81b0&l=404

```
-  FAIL - HIGH severity: missing_group_help
    Command-Group: `ml computetarget amlcompute` - Missing help
```

This failure is exposed by the change from https://github.com/Azure/azure-cli-dev-tools/pull/308.

`ml computetarget amlcompute` is missing help message, but it is not checked before https://github.com/Azure/azure-cli-dev-tools/pull/308:

```
> az ml computetarget amlcompute -h

Group
    az ml computetarget amlcompute

Subgroups:
    identity : AzureML compute target identity commands.
```

Only its subgroup `ml computetarget amlcompute identity` is checked:

```
> az ml computetarget amlcompute identity -h

Group
    az ml computetarget amlcompute identity : AzureML compute target identity commands.

Commands:
    assign : Assign identity to an AzureML compute target.
    remove : Remove identity from an AzureML compute target.
    show   : Display identities of an AzureML compute target.
```

The error should be fixed by adding help message to `ml computetarget amlcompute`.

**Pull request merged**